src/toil_scripts/tools/preprocessing.py

```diff
@@ -76,13 +83,61 @@ def run_samtools_index(job, bam_id):
     job.fileStore.readGlobalFile(bam_id, os.path.join(work_dir, 'sample.bam'))
     # Call: index the bam
     parameters = ['index', '/data/sample.bam']
-    docker_call(work_dir=work_dir, parameters=parameters,
+    docker_call(work_dir=work_dir,
+                parameters=parameters,
                 tool='quay.io/ucsc_cgl/samtools:0.1.19--dd5ac549b95eb3e5d166a5e310417ef13651994e')
     # Write to fileStore
     return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'sample.bam.bai'))
 
 
-def run_picard_create_sequence_dictionary(job, ref_id):
+def run_samtools_view(job, bam_id, ncores=1, flag='0'):
```

Comment:
We're just using this to filter out dupe reads, right? I'm dropping a line note here, because I think we should remove said filter. If we remove the filter, we should remove this as well.